docs/sources/flow/getting-started/migrating-from-operator.md

```diff
@@ -0,0 +1,128 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/getting-started/migrating-from-operator/
+description: Migrating from Grafana Agent Operator to Grafana Agent Flow
+menuTitle: Migrate from Operator
+title: Migrating from Grafana Agent Operator to Grafana Agent Flow
+weight: 320
+---
+
+# Migrating from Grafana Agent Operator to Grafana Agent Flow
+
+With the release of flow, Grafana Agent Operator is no longer the recommended way to deploy Grafana Agent in Kubernetes. Some of the Operator functionality has been moved into Grafana Agent
+itself, and the remaining functionality has been replaced by our Helm Chart.
+
+- The Monitor types (`PodMonitor`, `ServiceMonitor`, `Probe`, and `LogsInstance`) are all supported natively by Grafana Agent in Flow mode. You no longer are 
```

Comment:
I wonder if this should be a snippet that we could include also in the index page of the operator.

Reply:
Great idea, but probably slightly different wording. Will discuss seperately.